### DISCUSSION TITLE
Web hosting recommendations

### DISCUSSION THREAD

**Discussion by SueBanks** [datafeedr/wordpress-plugins#8] [Help]
Hi - I am looking for web hosting recommendations.

My current host has switched from CPanel to their own in house Site Tools, which among other things, allows their techs to log directly into my Wordpress sites without prior consent. One of my sites broke during their transition, and to confirm they had fixed the problem they took a screenshot from inside my Wordpress dashboard - which is the only reason I know.

Ideally I am looking for reseller hosting with CPanel, although this is not mandatory, and a non-EIG hosting company. 

Eric and Stefan - if this question is not the appropriate, please feel free to remove.

Thanks,

Susan

**Comment by EricBusch** [comment]
Hey Susan

Thanks for posting in our new forums!

This is getting tougher and tougher I think.

Both Stefan and I used LiquidWeb for many years. I think Stefan is still with them (though I'm not 100% on that). I was with them for about 10 years and was always very happy with their service and support. I've moved on to hosting on Digital Ocean for all of my sites now. Unfortunately no cPanel there. But if you want more info, I can share.

LiquidWeb does have cPanel/reseller plans: https://www.liquidweb.com/partner-community/reseller-hosting/

If the price isn't out of your price range, I would recommend them. 

Hope this helps!
Eric

**Comment by SueBanks** [comment]
Hi Eric!

Thanks for the suggestions, I will check out Liquid Web. I am not sure I even need CPanel or reseller hosting - but I have been using reseller hosting for my own sites for years and I am familiar with it. 

Yes, it is getting hard to find a good web host! 

thanks,

Susan

**Reply by EricBusch** [reply]
Hi Susan

If you don't need cPanel or Reseller hosting, then I would recommend SpinupWP and DigitalOcean.

The problem with cPanel is they drove up their prices for web hosts back in 2019 (I think). At that time I was on dedicated server at LiquidWeb which had cPanel. Because of cPanel's price spike, LW was going to be charging me more.

I knew I no longer needed a dedicated server and did not want to pay any more than what I was already paying. So I I moved all of my WordPress websites to SpinupWP and Digital Ocean just over a year ago. I've been pretty satisified. A bonus of using SpinupWP is that they have some really [good page caching](https://spinupwp.com/doc/caching-in-spinupwp/) technology which really helps reduce my costs at DigitalOcean.

> ### [UPDATE - March 31, 2022]
>
> If you do sign up to SpinupWP and you configure [automated backups](https://spinupwp.com/doc/site-backups/), be sure they are actually being created and are accessible. 
>
> My SpinupWP dashboard indicated that I had 30 days of backups on Amazon S3 for my GearGrabber site. However, last week, the DigitalOcean server which hosted my GearGrabber website crashed. No problem I thought, I'll just download a backup from S3 and restore on a new server. But when I logged into Amazon S3, there was only one backup available and that was from two years ago! This made restoring my website much more complicated because I needed to restore from a 6-day old DigitalOcean backup.  
> 
> SpinupWP's conclusion via their support email was:
>
> > _According to our logs, your backups were successfully uploaded to your S3 bucket. However, I noticed that the backup size is 2GB+, so the only thing that could happen was that the process timed out although it was recorded as successful. Unlikely, but possible._
> 
> Their reply suggests they do not actually believe this could happen and offered no apologies for not performing the service you pay them to do.
> 
> So if you do use SpinupWP's backup service, be very sure that your backups are in place and are being created at the interval you configured especially if you have a few thousand products in your WooCommerce store.

Anyway, that's what I've been using since Nov 2019. It's been ~~good~~ _OK_. GearGrabber runs there: [geargrabber.net](https://www.geargrabber.net/)

If you want to try that set up, here are some codes you could use which will save you $150 if you decide to stay with each service.

- [SpinupWP](https://spinupwp.com/?referral=x1jLAp2rDy) - Everyone that signs up with this referral link will get **$50 in credit** 30 days after signing up.
- [DigitalOcean](https://m.do.co/c/f35d2df4b9bc) - Everyone that signs up with this referral link gets **$100 in credit** over 60 days.

Of course those are my affiliate links but they also benefit you.

Let me know if you have any questions.

Eric

**Reply by SueBanks** [reply]
Thanks for the suggestion Eric!  I am setting up SpinWP on Digital Ocean right now, with the help of my son @jeremybanks

**Reply by EricBusch** [reply]
Cool, let me know how it goes and what you think!

**Reply by SueBanks** [reply]
Hi Eric,

First blank WP demo site up, and it loads very fast! I haven't installed any themes, plugins or content yet.

I did have a couple of questions for you.
1. Do you offload your images files offsite? I started using Short Pixel's CDN for my API site and since the original files are backed up, the WP install/uploads folder is still very large, which makes a very large backup.
2. Related to above, I normally use backup buddy to backup my sites and send them to their offsite Stash. I noticed that there is a backup option on SpinupWP. Just wondering if you use that.

Thanks!

Susan

**Reply by EricBusch** [reply]
That's cool!

1. No, unfortunately I do not offload my images on any of my WordPress sites except for one. In that case I use WP Offload Media (developed by the same folks that created SpinupWP).

2. There **_is_** a backup option for SpinupWP and it's integrated with Amazon S3. You can configure your backups per site. If you have an S3 account it's super easy. You can also restore right from the SpinupWP interface. https://spinupwp.com/doc/site-backups/

Hope this helps!

Eric

**Reply by SueBanks** [reply]
Hi Eric,

Do you like WP Offload media?

I saw that SpinupWP had a backup option. It also integrates with Digital Ocean too. :)

Thanks,

Susan

**Reply by EricBusch** [reply]
Yes, I like WP Offload media. Works well so far and keeps backups very small.

**Reply by SueBanks** [reply]
Great  - thanks! :)
